src/lib.rs

```diff
@@ -545,6 +545,15 @@ mod test {
         assert_eq!(tip_hash, tip_hash_async);
     }
 
+    #[cfg(all(feature = "blocking", any(feature = "async", feature = "async-https")))]
+    #[tokio::test]
+    async fn test_get_block_hash() {
+        let (blocking_client, async_client) = setup_clients().await;
+        let block_hash = blocking_client.get_block_hash(21).unwrap();
+        let block_hash_async = async_client.get_block_hash(21).await.unwrap();
+        assert_eq!(block_hash, block_hash_async);
```

Comment:
You may want to add another check comparing the Esplora results to RPC, i.e., `BITCOIND.client.get_block_hash(21)`

Reply:
That's right. Added: https://github.com/bitcoindevkit/rust-esplora-client/commit/7c66cf1f0058feb11bff3bacad98068bfd1bbf82